docs/source/backends.rst

```diff
@@ -137,6 +138,31 @@ torch.backends.openmp
 .. py:module:: torch.backends.xnnpack
 
 
+torch.backends.opt_einsum
+^^^^^^^^^^^^^^^^^^^^^^^^
+.. automodule:: torch.backends.opt_einsum
+
+.. autofunction:: torch.backends.opt_einsum.is_available
+
+.. autofunction:: torch.backends.opt_einsum.get_opt_einsum
+
+.. attribute::  torch.backends.opt_einsum.enabled
+
+    A :class:`bool` that controls whether opt_einsum is enabled (`True` by default). If so,
```

Comment:
btw if you want to make something appear like code, you'll need to put double ` ``backticks`` ` otherwise it will just make it italic